test/spec/smartbanner_spec.js

```diff
@@ -224,6 +225,76 @@ describe('SmartBanner', function() {
 
     });
 
+    context('when enabledPlatform set to android, but is openned on a iPhone', function() {
```

Comment:
Let's improve here a bit, e.g. `when enabled-platform set to android, but opened on iOS`